imports.go

```diff
@@ -11,14 +11,25 @@ import (
 	"strings"
 )
 
+type listOutput struct {
+	Imports     []string
+	Deps        []string
+	TestImports []string
+}
+
 func parseImports(recursive bool, testDependencies bool) ([]string, error) {
 	var imports []string
 	packages, err := listPackages()
 	if err != nil {
 		return imports, fmt.Errorf("error listing packages: %s", err)
 	}
 
-	ensureDependenciesExist(packages, testDependencies)
+	// Ensuring our dependencies exists isn't a strict requirement, therefore
+	// only print a message to stderr rather then completely failing
+	err = ensureDependenciesExist(packages, testDependencies)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s", err)
```

Comment:
here should also return error on top level and print it in place where function parseImports has called